easybuild/easyconfigs/l/libcerf/libcerf-2.4-GCC-13.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'libcerf'
+version = '2.4'
+
+homepage = 'https://jugit.fz-juelich.de/mlz/libcerf'
+
+description = """
+ libcerf is a self-contained numeric library that provides an efficient and
+ accurate implementation of complex error functions, along with Dawson,
+ Faddeeva, and Voigt functions.
+"""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v%(version)s/']
+sources = ['libcerf-v%(version)s.tar.gz']
+checksums = ['080b30ae564c3dabe3b89264522adaf5647ec754021572bee54929697b276cdc']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.27.6'),
+    ('Perl', '5.38.0'),  # required for pod2html
+]
+
+installopts = ' && $FC $FCFLAGS -c %(builddir)s/%(name)s-v%(version)s/fortran/ccerflib_f95_interface/use_libcerf_mod.f90 '
```

Comment:
```suggestion
installopts = ' && $FC $FCFLAGS -c %(start_dir)s/fortran/ccerflib_f95_interface/use_libcerf_mod.f90 '
```

Reply:
Thanks @jfgrimm, I wasn't aware of this template.